AIPM-Registry/DATA_MODEL_AND_SQL.md

```
@@ -0,0 +1,275 @@
+  tags TEXT[],                      -
+
+- freeform tags
+
+  semantics_hash TEXT,              -
+
+- hash of normalized I/O semantics (for cross-provider matching)
+
+  created_at TIMESTAMPTZ DEFAULT now()
+);
+
+- - Concrete implementations (variant = functio
```

markdownlint-cli2 (0.18.1)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)